spectree/utils.py

```diff
@@ -44,14 +44,17 @@ def parse_params(func, params, models):
     get spec for (query, headers, cookies)
     """
     if hasattr(func, 'query'):
-        params.append({
-            'name': func.query,
-            'in': 'query',
-            'required': True,
-            'schema': {
-                '$ref': f'#/components/schemas/{func.query}',
-            }
-        })
+        for query_param in models[func.query]["properties"].keys():
```

Comment:
How about iterating over `.items()` instead of just `.keys()`? You can then replace `models[func.query]["properties"][query_param]` below just with the value from key/value pair you iterate over. Feels easier to read.

Reply:
check the current implementation. https://github.com/0b01001001/spectree/blob/49d85ec20b7c3756a774c10e32d7be248a7b2a14/spectree/utils.py#L49-L57

This PR is covered by #49.

I'll close it.